<?xml version="1.0"?>

<launch>
<!--	
<include file="$(find turtlebot3_gazebo)/launch/turtlebot3_world.launch"/>
 -->
 <node name="navigate_server"
    pkg="simple_navigation_goals"
    type="navigate_server.py"
    output="screen"/>

     <node name="trash_can_server"
    pkg="simple_navigation_goals"
    type="trash_can_server.py"
    output="screen"/>
 
</launch>
